### ISSUE TITLE
`zones` variable in `confluent_network` resource too restrictive in terms of min/max AZs

### ISSUE THREAD

**Issue by Tarun-S** [confluentinc/terraform-provider-confluent#270]
The `zones` variable for the `confluent_network` resource currently supports exactly three availability zones in the list given as input. However, at least for AWS-based networks, there can be more than three AZs. For example, I'm working with a cluster that has 5 AZs. Inputting 5 AZs results in the following plan error: 

```
Error: Too many list items

<terraform code in question>

Attribute supports 3 item maximum, but config has 5 declared.
```

I've imported the `confluent_network` using `terraform import`, with the `zone` variable in my Terraform code removed, and the plan shows a diff, with the main difference being the 5 missing AZs from the `zone` variable.

I'm not sure what Confluent Cloud supports for networks, in terms of the minimum or maximum number of AZs. Whatever that range is, the min/max values should be updated from the current value of 3 AZs.

If I can get some information on what this range should be, I can try throwing up a quick PR.

**Comment by Tarun-S** [comment]
I'm in contact with Confluent Cloud support, and it looks like the Network I'm trying to import may be in an anomalous state, as it shouldn't have 5 AZs attached to it. So it might not be an issue with the Provider itself but just something wrong with the Network. I'll post an update when I know for sure.

**Comment by linouk23** [comment]
@Tarun-S I'm not a networking expert but it seems like Networking API only supports [3 zones](https://docs.confluent.io/cloud/current/networking/private-links/aws-privatelink.html).

Could you share where you run into 5 zones?

**Comment by Tarun-S** [comment]
@linouk23 It looks like this was because the Network I was trying to import somehow had 5 AZs attached to it. After speaking with Confluent Cloud support, it seems that Networks must have 3 AZs attached to them, and the Network I had was somehow misconfigured. After support removed two AZs from the Network, I was able to successfully import it :) So it looks like there's nothing wrong with the provider itself.

**Comment by linouk23** [comment]
This was fixed in `1.49.0`: https://github.com/confluentinc/terraform-provider-confluent/blob/master/CHANGELOG.md#1490-july-17-2023
